aspnetcore/test/middleware.md

```diff
@@ -31,17 +31,14 @@ In the test project, create a test:
 
 * Build and start a host that uses <xref:Microsoft.AspNetCore.TestHost.TestServer>.
 * Add any required services that the middleware uses.
-* Add the [Microsoft.AspNetCore.TestHost](https://www.nuget.org/packages/Microsoft.AspNetCore.TestHost/) NuGet package to the project:
-  
-  ```dotnetcli
-  <ItemGroup>
-    <PackageReference Include="Microsoft.AspNetCore.TestHost" Version="3.1.*" />
-  </ItemGroup>
-  ```
+* Add package references to the project for the following NuGet packages:
+
+  * [Microsoft.AspNetCore.TestHost](https://www.nuget.org/packages/Microsoft.AspNetCore.TestHost/)
+  * [Microsoft.AspNetCore.Hosting](https://www.nuget.org/packages/Microsoft.AspNetCore.Hosting/)
```

Comment:
The Microsoft.AspNetCore.Hosting package is obsolete since 3.0, that library is now only in the shared framework.

Reply:
Maybe but your sample with Configure didn't work even using net6 LTS and only nuget package you mentioned
I had to add this nuget to have it working

Reply:
If obsolete @Tratcher which library is replacement for those extension methods?